Cargo.toml

```diff
@@ -24,4 +24,5 @@ structopt = "0.3"
 signal-hook = "0.1"
 reqwest = { version = "0.10", features = ["blocking"] }
 get-port = "1.3.1"
-lazy_static = "1.4.0"
\ No newline at end of file
+lazy_static = "1.4.0"
```

Comment:
Where is this crate used ?

Reply:
this crate was already there

Reply:
Ok it's a miss going to remove it on a cleanup PR :+1: